Assets/Scripts/Combat/Abilities/AbilityDescriptions/BonkDescription.asset

```diff
@@ -12,5 +12,5 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: fd2e78d0aff466c449c00d1841840feb, type: 3}
   m_Name: BonkDescription
   m_EditorClassIdentifier: 
-  Name: Bonk
+  Name: 'Bonk - Candy Cost: 0'
```

Comment:
Can we make the cost here dynamic? Adjusting to the actual cost? Might be overkill if it's tricky to do, but I can see a bug coming when the cost here !== cost on the ability. 

On that note, maybe decouple the cost from this description, and instead have a lil candy corn cost UI bubble that dynamically fetches the current ability's cost...?

Reply:
Let's revisit this haha